bundles/org.eclipse.equinox.log.stream/META-INF/MANIFEST.MF

```diff
@@ -3,13 +3,14 @@ Bundle-ManifestVersion: 2
 Bundle-Name: %bundleName
 Bundle-Vendor: %bundleVendor
 Bundle-SymbolicName: org.eclipse.equinox.log.stream
-Bundle-Version: 1.0.400.qualifier
+Bundle-Version: 1.0.500.qualifier
 Bundle-Activator: org.eclipse.equinox.internal.log.stream.LogStreamManager
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-Localization: plugin
 Import-Package: org.osgi.framework;version="[1.9.0,2.0.0)",
  org.osgi.service.log;version="[1.4.0,2.0.0)",
  org.osgi.service.log.stream;version="[1.0,1.1)",
+ org.osgi.util.function;version="[1.1.0,2.0.0)",
```

Comment:
If you are importing `promise` at `1.0`, then you should also import function at `1.0` to allow for resolving in systems which are using the original promise/function bundles. The log stream OSGi API bundle and spec only require version 1.0 of promise/function.

So far function and promise versions have been in sync. This could of course change in the future.

Reply:
Thanks for the hint.
In the (hopefully near) future these package Imports will be removed via https://github.com/eclipse-equinox/equinox.bundles/pull/26. Therefore I think we don't have to adjust that now.